controls/datagrid/grouping/overview.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Overview
+page_title: .NET MAUI DataGrid Documentation - Grouping Overview
+description: "Review the Telerik .NET MAUI DataGrid Grouping documentation article to learn more about all built in Grouping functions you can use."
+position: 0
+slug: datagrid-grouping
+---
+
+# Overview
+
+`Telerik UI for .NET MAUI DataGrid` provides a programmatic as well as a drag and drop approach for grouping its data based on a specific criteria
+
+## Grouping UI
+
```

Comment:
suggestion: The DataGrid Grouping UI is enabled by design and it allows user to group the data by column value. The UI is represented by the  `DataGridGroupingPanel`. The panel is part of the DataGrid and it is visualized at the top. 
You can group the DataGrid by dragging and dropping the column headers to the `DataGridGroupingPanel`.